client/Combatant/StaticCombatantViewModel.ts

```diff
@@ -17,6 +18,7 @@ module ImprovedInitiative {
             this.Initiative = combatant.Initiative();
             this.IsPlayerCharacter = combatant.IsPlayerCharacter;
             this.Tags = combatant.Tags();
+            this.AllowSuggestions = Store.Load(Store.User, "PlayerViewAllowPlayerSuggestions");
```

Comment:
This won't work as expected when the player view URL is accessed on a different machine. The AllowSuggestions flag needs to be passed as part of the Saved Encounter payload. This can be done similarly to the way the 'DisplayTurnTimer' flag is sent.